docs/source/release_notes.rst

```diff
@@ -2,6 +2,7 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
+        * Make utils to handle the logic for threshold tuning objective and resplitting data :pr:`3888`
```

Comment:
ah damn! I had noticed that and had been doing past tense to fit in. I'll make the change!

For what it's worth, my commit message (and release note since I mostly treat them the same) writing style comes from [this article](https://cbea.ms/git-commit/) that I was ~~forced~~ asked to read at my first internship. The thing that made the biggest impression on me was this section 
![image](https://user-images.githubusercontent.com/64278226/207903154-ebac5505-54f4-4364-92fc-217f80bc32cc.png)

Reply:
okay, I see the internet is kind of split between what tense release notes should be in, but more of them say past tense than present tense.

Reply:
Yeah, my personal ethos has always been past tense for release notes and present tense for commits, and that seems to be what we generally follow. Super interesting article though, lots to think about there!